src/ctree-app/ctree-app.html

```diff
@@ -185,6 +190,11 @@
         float: right;
         margin: 4px;
       }
+      paper-toggle-button.green {
+        --paper-toggle-button-checked-bar-color:  var(--paper-green-500);
+        --paper-toggle-button-checked-button-color:  var(--paper-green-500);
```

Comment:
Please try to keep consistent code formatting (there are 2 spaces before `var` instead of 1)